Hue_low : 24 
Hue_high : 175 
Saturation_low : 1 
Saturation_high : 255 
Value_low : 2 
Value_high : 180 
gray_thresh : 133 
number_of_lines : 29 
error_threshold : 0.21 
Width_min : 20 
Width_max : 83 
inverted_filter : 1 
camera_start_height : 312 
camera_bottom_height : 630 
camera_left_width : 160 
camera_right_width : 640 
steering_sensitivity : 1.0 
zero_throttle : 0.125 
no_error_throttle : 0.15200000000000014 
error_throttle : 0.14200000000000013 
